plugins/joblib_launcher/setup.py

```diff
@@ -0,0 +1,25 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+from setuptools import find_packages, setup
+
+with open("README.md", "r") as fh:
+    LONG_DESC = fh.read()
+    setup(
+        name="joblib-launcher",
```

Comment:
Please call this package `hydra-joblib-launcher`, I would like all Hydra plugins to have the hydra- prefix for better discoverability.